### ISSUE TITLE
The new installer is not suitable for use with a non-Latin layout

### ISSUE THREAD

**Issue by PlatonB** [elementary/installer#513]
**Describe the bug**
The bug is widespread in non-Ubiquity and non-Anaconda installers. Unfortunately, it also affected elementary OS.
- The bug is well described and already fixed in [Calamares](https://github.com/calamares/calamares/issues/1211).
- There is also a description with screenshots in the [Pop!_OS](https://github.com/pop-os/gnome-initial-setup/issues/43).
- The Pop!_OS bug comes from [gnome-initial-setup](https://gitlab.gnome.org/GNOME/gnome-initial-setup/-/issues/103).

**To Reproduce**
1. Select the Cyrillic layout.
2. Try to enter your login and password.

**Comment by danrabbit** [comment]
The Installer doesn't handle user creation which is what I'm assuming you mean by "login and password". Are you perhaps referring to initial setup which happens after you install and reboot?

**Comment by MrPotatoBobx** [comment]
With arabic, you can set "Full name" and password with non latin characters, but the user cannot be in arabic
Is this some limitation with linux?
![image](https://user-images.githubusercontent.com/63403023/116787346-28c49200-aaac-11eb-81c3-8cd0197c878d.png)

**Comment by MrPotatoBobx** [comment]
> The Installer doesn't handle user creation which is what I'm assuming you mean by "login and password". Are you perhaps referring to initial setup which happens after you install and reboot?

Im guessing it can also affect the encyrption section, i think you should move this to https://github.com/elementary/initial-setup

**Comment by davidmhewitt** [comment]
So what happens with the disk encryption password? Is it valid/expected to create a disk encryption password with non-latin characters? And does that password work when booting and decrypting the disk?

I'm just checking whether there are two issues here. If not, we can move this to initial setup.

**Comment by MrPotatoBobx** [comment]
> So what happens with the disk encryption password? Is it valid/expected to create a disk encryption password with non-latin characters? And does that password work when booting and decrypting the disk?
> 
> I'm just checking whether there are two issues here. If not, we can move this to initial setup.

1. Yea you can put non Latin passwords in the drive encryption thing
2. I'll have to reinstall to check that, please be a bit patient

I think it only effects the username part in user creation

**Comment by MrPotatoBobx** [comment]
Decrypts and boots up fine if the password has non latin characters

**Comment by PlatonB** [comment]
Yes, I forgot that creating username and password is now part of a separate program. But the bug is actual.
